meson.build

```diff
@@ -10,7 +10,7 @@
 #		 considered. Yet, consumer must be recompiled.
 # Patchlevel Version: API / ABI compatible, no functional changes, no
 #		      enhancements, bug fixes only.
-project('esdm', 'c',
+project('esdm', ['c', 'cpp'],
```

Comment:
Is it not possible to add this to get_option('botan-rng').enabled() as separate calls to add_global_arguments?

Reply:
It seems to be not possible with add_global_arguments (no compiler is searched). add_languages seems to do the trick. I'll add this in an updated PR.